transmart-schemas/rebuild.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+here=$(dirname "${0}")
+version=$(cat "${here}/VERSION")
+jar="${here}/build/libs/transmart-schemas-${version}.jar"
+
+# Check if the jar present
+if [[ ! -e "${jar}" ]]; then
+    echo "Cannot find jar: ${jar}"
+    echo "To build it, run:"
+    echo "  gradle assemble"
+    exit 1
+fi
+
+# Choose the location of the java binary
+prefix=
+if [[ ! "x${JAVA_HOME}" == "x" ]]; then
+    prefix="${JAVA_HOME}/bin/"
+fi
+
+
+sudo -u postgres psql -c 'drop database "transmartDev";'
+sudo -u postgres psql -c 'create database "transmartDev";'
+sudo -u postgres psql -c 'grant all on database "transmartDev" to tm_admin;'
+${prefix}java -jar -Dspring.config.location=./database.yml "${jar}"
+sudo -u postgres psql -d transmartDev -c 'grant all on schema i2b2demodata to biomart_user;'
```

Comment:
Shouldn't `biomart_user` and `tm_cz` be created first?

Reply:
Yes, but there is no `create role ... if not exists`... I will try some other way